WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -234,19 +238,34 @@ private extension PaymentCaptureOrchestrator {
                                  metadata: metadata)
     }
 
+    private func synchronizePlugins(from siteID: Int64) async throws {
+        return try await withCheckedThrowingContinuation({
+                (continuation: CheckedContinuation<(), Error>) in
+            let systemPluginsAction = SystemStatusAction.synchronizeSystemPlugins(siteID: siteID) { result in
+                if case let .failure(error) = result {
+                    DDLogError("[PaymentCaptureOrchestrator] Error syncing system plugins: \(error)")
+                    continuation.resume(throwing: error)
+                } else {
+                    continuation.resume(returning: ())
+                }
+            }
+            ServiceLocator.stores.dispatch(systemPluginsAction)
```

Comment:
Yeah, that makes sense, I will update after I test it to double-check that it happened already

Reply:
After some checking I think it is better to leave this synching. When paying directly in the orders tab without going through the settings onboarding we do not synch the plugins. We do when we start the app, but It could be that the plugins state changed during the app execution.